gabbi/case.py

```diff
@@ -402,7 +406,34 @@ def _test_status(self, expected_status, observed_status):
             statii = [stat.strip() for stat in expected_status.split('||')]
         else:
             statii = [expected_status.strip()]
-        self.assertIn(observed_status, statii)
+
+        self.assert_in_or_print_output(observed_status, statii)
+
+    def assert_in_or_print_output(self, expected, iterable):
+        if self.content_type != "application/binary":
```

Comment:
(I made this comment already but it seems to have been swallowed somewhere, so if you've seen this already, my apologies.)

Maybe use [gabbi.utils:not_binary](https://github.com/cdent/gabbi/blob/master/gabbi/utils.py#L82) here so that things like `image/png` also get skipped.